mlflow/pyspark/ml/__init__.py

```diff
@@ -85,7 +85,7 @@ def _parse_allowlist_file(line_iter):
             username=url_parsed.username,
             password=url_parsed.password,
         )
-        response = http_request(host_creds=host_creds, endpoint=path, method="GET")
+        response = http_request(host_creds=host_creds, endpoint=path, method="GET", stream=True)
         augmented_raise_for_status(response)
         return _parse_allowlist_file(response.iter_lines(decode_unicode=True))
```

Comment:
`iter_lines` is no-op if the response is already closed. `stream=True` needs to be added.